src/controllers/index.js

```diff
@@ -309,8 +309,10 @@ Controllers.manifest = function (req, res) {
 			density: 4.0,
 		});
 	}
+	plugins.fireHook('filter:manifest.build', manifest, function (err, manifest) {
+                res.status(200).json(manifest);
```

Comment:
need to check err here and pass it to next.
```
if (err) {
     return next(err);
}
```

Reply:
Thanks, fixed it :)